markdown/bitburner.corporation.createcorporation.md

```diff
@@ -17,11 +17,17 @@ createCorporation(corporationName: string, selfFund: boolean): boolean;
 |  Parameter | Type | Description |
 |  --- | --- | --- |
 |  corporationName | string | Name of the corporation |
-|  selfFund | boolean | If you should self fund, defaults to true, false will only work on BitNode 3 |
+|  selfFund | boolean | If you want to self-fund. Defaults to true, false will only work in BitNode 3. |
 
 **Returns:**
 
 boolean
 
 true if created and false if not
 
+## Remarks
+
+RAM cost: 20 GB
+
+This function throws an error if: - Try to self-fund outside BitNode 3. - Be in a BitNode that has CorporationSoftcap (a BN modifier) less than 0.15. Use [getBitNodeMultipliers](./bitburner.ns.getbitnodemultipliers.md) to get the value of this modifier.
```

Comment:
Bulleted lists get munged

Reply:
Regarding CorporationSoftcap,

The game uses `CorporationSoftcap < 0.15` to mark that corp is disabled in that bn. (Currently, only bn8 has CorporationSoftcap === 0).
See also https://github.com/bitburner-official/bitburner-src/blob/633da383016ad0521f9f1c17cdd99478d2701e41/src/BitNode/ui/BitnodeMultipliersDescription.tsx#L333-L339

Maybe number 0.15 is here because they want to prevent "tax" formular breakout when "tax exponent" goes to negative.

However, I think `CorporationSoftcap < 0.15` is implementation details, and there should be some method to provide whether Corporation is disabled in this bn without refering CorporationSoftcap. (accessing CorporationSoftcap needs sf5)

Reply:
You can try to create a corporation and see if it succeeds. :D

Reply:
I agree with bupjae:
- Accessing CorporationSoftcap requires SF5.
- Need to know the "magic" number 0.15 (by reading the TSDoc). Implementation details like this should not matter for players.
- Technically, they can bypass this problem by wrapping the API in try-catch, but it looks hacky and ugly.

@d0sboots Can I add `ns.corporation.isCorporationDisabled`?

Reply:
I'd call it isCorporationEnabled or (perhaps better) canCreateCorporation, which potentially could test for more cases. 

But yeah, go for it if you want.